### DISCUSSION TITLE
Welcome to scikit-extremes Discussions!

### DISCUSSION THREAD

**Discussion by kikocorreoso** [kikocorreoso/scikit-extremes#10] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by NoorAlMahmud** [comment]
Hi, I am a research intern, using sk-extremes package to analyze wind speed and rainfall data from cyclone simulations. I mainly use the distribution classes in classics. The GEV and Gumbel are fine. But, it seems that GPD class is incomplete (though it extends the base class). I can't create any GPD (Generalized Pareto Distribution) objects. Does this package not support GPD or am I doing something wrong?

I would really appreciate your input. (Btw, your this package has been very helpful in my research work)

--Noor

**Reply by kikocorreoso** [reply]
Hi,

If you read [here](https://github.com/kikocorreoso/scikit-extremes#to-do) it says GPD is in the TO-DO list. 

I'm sorry but these days I do not have much time to implement more functionalities but I'm open to review pull requests in case you want to push some code.

KR,
